### ISSUE TITLE
Blocking 120's

### ISSUE THREAD

**Issue by jabin150** [funkydude/BadBoy_Levels#9]
I have my levels set at 100 for dh 60 for dk and 5 for others, yet it's blocking all 120's from whispering me.

**Comment by arcticnw** [comment]
I've had the same issue. A random 120 told me, that it replies to them, that message has been blocked as they aren't level 3 yet.

**Comment by L1ghtelf** [comment]
Same thing. My friend can't whisper me as lvl 120 with this plugin. He even tried to whisper me from different characters and he still was blocked. This plugin is currently broken unfortunately.

**Comment by funkydude** [comment]
This is not something I can reproduce. I recommend trying with all non-BadBoy addons disabled and seeing if the issue still persists.

Alternatively, if the issue is now gone due to the level squish, let me know please.

**Comment by L1ghtelf** [comment]
The issue still exists even with level squish. Currently I have 100% reproduction rate of this bug because it seems I can't whisper to anyone who has this plugin installed as level 50. We tested this with all addons disabled.

Proof:
![image](https://user-images.githubusercontent.com/16833/96052149-d788e400-0e74-11eb-9ea6-61176c1863b4.png)

**Comment by funkydude** [comment]
Thanks for the extra info.
I see you had all your addons disabled, but did the person you were whispering have all non-BadBoy addons disabled?

edit: Also tell me the following things: Is the person 1) In your guild, 2) On your basic character friends list, 3) On your BNet friends list

**Comment by L1ghtelf** [comment]
The person that I was whispering had all addons disabled except BadBoy. That guy is not in my guild and he is not in my basic or Bnet friends list. 
Here is the thing: when I whispered this guy, he didn't get any message, but he got the notification that "[Arcticforce] has come online", which is very odd since he doesn't have me in his friend list or guild.

This thing is not a first occurrence btw. I got blocked by 4 or 5 people since August . Changing characters doesn't work. It feels like my whole account got blacklisted by everyone who uses this plugin :P
